locations/spiders/livique_ch.py

```diff
@@ -0,0 +1,68 @@
+import re
+from scrapy.spiders import SitemapSpider
+
+from locations.hours import DAYS_DE, OpeningHours
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class LiviqueCHSpider(SitemapSpider, StructuredDataSpider):
+    name = "livique_ch"
+    brands = {
+        "Livique": "Q15851221",
+        "Lumimart": "Q111722218",
+    }
+    allowed_domains = ["www.livique.ch"]
+    sitemap_urls = ["https://www.livique.ch/sitemap.xml"]
```

Comment:
```suggestion
    sitemap_urls = ["https://www.livique.ch/sitemap.xml"]
    sitemap_follow = ["/STORE-"]
```
We can avoid pulling the product/category sitemaps with the above.